balancer-js/src/types.ts

```diff
@@ -126,30 +127,31 @@ export interface BalancerNetworkConfig {
 }
 
 export interface BalancerDataRepositories {
+  // Replace with a swapFeeRepository, we don't need historic pools for any other reason than to get the swap fee
```

Comment:
I think this comment was specifically related to `yesterdaysPools` and moving it here means we've lost that context.